### PR TITLE
Designdoc blockdev updates

### PR DESCRIPTION
**Pull request by agrover**

### CODE REVIEW

**Comment by agrover** [comment]
screenshot from 2017-08-31 11-30-35

File: docs/design/StratisSoftwareDesign.lyx [hunk 1]
```diff
@@ -2302,7 +2302,7 @@ description
 \begin_inset Text
 
 \begin_layout Plain Layout
-IEEE CRC32 of signature block (bytes at offset 4 length 508)
+IEEE CRC32C of signature block (bytes at offset 4 length 508)
```
Comment (mulkieran) [inline docs/design/StratisSoftwareDesign.lyx]: We will need to change implementation to reflect this change.

Comment (agrover) [reply docs/design/StratisSoftwareDesign.lyx]: I assumed we already were using crc32c? But if not, then yes. Also, there's a new field in static header we will need to read/write.

Comment (mulkieran) [reply docs/design/StratisSoftwareDesign.lyx]: Nope, we are using ieee.

Comment (mulkieran) [reply docs/design/StratisSoftwareDesign.lyx]: I will do it now, one fewer issue to write up.

**Pull request merged**